Generators/src/main/java/io/deephaven/pythonPreambles/KafkaToolsPreamble.txt

```diff
@@ -73,56 +74,58 @@ def _passThrough(wrapped, instance, args, kwargs):
 
 
 @_passThrough
-def _custom_avroSchemaToColumnDefinitions(*args):
-    if len(args) == 0:
-        raise Exception('not enough arguments')
-    if len(args) == 1:
-        return _java_type_.avroSchemaToColumnDefinitions(args[0])
-    if len(args) == 2:
-        schema = args[0]
-        dict = args[1];
-        fieldNamesArray = jpy.array('java.lang.String', dict.keys())
-        columnNamesArray = jpy.array('java.lang.String', dict.values())
-        mapping = _java_type_.fieldNameMappingFromParallelArrays(fieldNamesArray, columnNamesArray)
-        return _java_type_.avroSchemaToColumnDefinitions(schema, mapping)
-    raise Exception('too many arguments: ' + len(args))
+def _custom_avroSchemaToColumnDefinitions(schema, mapping:dict = None):
+    if mapping is None:
+        return _java_type_.avroSchemaToColumnDefinitions(schema)
+
+    if not isinstance(mapping, dict):
```

Comment:
Still necessary with that mapping:dict specifier?

Reply:
Yes, there is no enforcement from python, only warnings from linters/IDEs.